### PR TITLE
*: unify all integer types to int64 and uint64

### PR DESCRIPTION
**Pull request by coocood**
Simplify type related code.

### CODE REVIEW

File: column/column.go [hunk 1]
```diff
@@ -372,98 +372,99 @@ func (c *Col) castIntegerValue(val int64, errCode int) (casted interface{}, err
...
 			} else if val > 1<<24-1 || errCode == errCodeOverflowUpper {
 				overflow = true
-				casted = uint32(1<<24 - 1)
+				casted = uint64(1<<24 - 1)
 			} else {
-				casted = uint32(val)
+				casted = uint64(val)
 			}
 		} else {
 			if val > 1<<23-1 || errCode == errCodeOverflowUpper {
```
Comment (siddontang) [inline column/column.go]: we may define a const for MinInt24/MaxInt24/MaxUint24

Comment (qiuyesuifeng) [inline column/column.go]: +1

**Comment by siddontang** [comment]
LGTM

**Comment by qiuyesuifeng** [comment]
LGTM

**Pull request merged**